packages/protocol/contracts/L1/TaikoConfig.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.18;
+
+import {TaikoData} from "../L1/TaikoData.sol";
+
+library TaikoConfig {
+    function getConfig() internal pure returns (TaikoData.Config memory) {
+        return
+            TaikoData.Config({
+                chainId: 167,
+                // Two weeks if avg block time is 10 seconds
+                maxNumProposedBlocks: 120960,
+                ringBufferSize: 120960 + 10,
+                // Each time one more block is verified, there will be ~20k
+                // more gas cost.
+                maxVerificationsPerTx: 10,
+                // Set it to 6M, since its the upper limit of the Alpha-2
+                // testnet's circuits.
+                blockMaxGasLimit: 6000000,
+                // Set it to 79  (+1 TaikoL2.anchor transaction = 80),
+                // and 80 is the upper limit of the Alpha-2 testnet's circuits.
+                maxTransactionsPerBlock: 79,
+                minEthDepositsPerBlock: 8,
+                maxEthDepositsPerBlock: 32,
+                maxEthDepositAmount: 10000 ether,
+                minEthDepositAmount: 1 ether,
+                // Set it to 120KB, since 128KB is the upper size limit
+                // of a geth transaction, so using 120KB for the proposed
+                // transactions list calldata, 8K for the remaining tx fields.
+                maxBytesPerTxList: 120000,
+                minTxGasLimit: 21000,
+                proofCooldownPeriod: 5 minutes,
```

Comment:
For mainnet I think this will need to be much higher because this works in the same way as a fraudproof, so the same censorship concerns.

Reply:
Change it to 30 minutes?

Reply:
Still seems low to me, but I think the balance is the amount of money that can be stolen (up to the TVL of the rollup) and the cost to censor the oracle proof transaction on L1 for a certain period of time. So if the TVL is low the challenge period can be quite short, but if the TVL is high then probably needs a long window (like optimistic rollups).

Reply:
I see, but if this value is large, we need to find a liquidity bridge partner.

Reply:
I think for now, we can use 30 minutes for testnets.